unused-images-crawler/README.md

```diff
@@ -0,0 +1,16 @@
+# Script to get unused images and assets in Storyblok
+
+This is a small script you can use to get unused images and other assets from your content items and put them in a folder. If the folder does not exist it will be created during the process.
+
+## How to use
+
+1. Get your Oauth token from the My account section of Storyblok.
```

Comment:
Do you mean personal access token?

Reply:
@eunjae-lee  you are right, thanks for spotting this. @alexadark can you please change this too?